src/notice/notice.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
```

Comment:
At first, typeORM and @nestjs/typeorm were confused.

If you read the link below carefully, you'll need to take a Repository from a TypeORM, data source, and set it up as you inject dependencies.
But, @nestjs/typeform has wrapped this up and made it simple to use.

https://docs.nestjs.com/recipes/sql-typeorm